src/aws/cloudwatch-sink.cpp

```diff
@@ -327,13 +327,18 @@ std::string CloudWatchSink::log_stream_name(const Log& log, const Channel& chann
 
 std::string CloudWatchSink::formatted_json(Log const& log,
                                            Channel const& channel,
-                                           ContextInfo const& context_info) const
+                                           ContextInfo const& context_info,
+                                           ContextInfo const& global_context_info
+                                           ) const
 {
     nlohmann::json j;
     std::stringstream ss;
     std::time_t log_time_t = std::chrono::system_clock::to_time_t(log.time_created());
     struct tm timeinfo;
+    auto ms = std::chrono::duration_cast<std::chrono::milliseconds>(log.time_created().time_since_epoch());
+    auto fraction = ms.count() % 1000;
     ss << std::put_time(compat::localtime(&log_time_t, &timeinfo), "%FT%T%z");
+    ss << "." << std::setfill('0') << std::setw(3) << fraction;
```

Comment:
@aradyaron - I'm pretty sure the ms are in the wrong place.
(They are now part of the timezone instead of being part of the time)
I'm changing to:
```
    auto milliseconds = std::chrono::duration_cast<std::chrono::milliseconds>(log.time_created().time_since_epoch()) % 1000;
    struct tm timeinfo;
    ss << std::put_time(compat::localtime(&log_time, &timeinfo), "%FT%T");
    ss << '.' << std::setw(3) << std::setfill('0') << milliseconds.count();
    ss << std::put_time(compat::localtime(&log_time, &timeinfo), "%z");
```